doc/content/verification_and_validation/ver-1c.md

```diff
@@ -2,38 +2,37 @@
 
 # Diffusion Problem with Partially Preloaded Slab
 
-This verification problem is taken from [!cite](longhurst1992verification). Diffusion of tritium through a semi-infinite SiC layer is modeled with an initial
-loading of 1 atom/m{^3} in the first 10 m of a 2275-m slab. Diffusivity is unity
+This verification problem is taken from [!cite](longhurst1992verification,ambrosek2008verification). Diffusion of tritium through a semi-infinite SiC layer is modeled with an initial
+loading of 1 atom/m$^3$ in the first 10 m of a 100 m slab. TMAP4 uses a slab length of 2275 m, however using a smaller slab length was found to not change the results. Additionally, the smaller domain size allows getting a finer simulation mesh for the same computational cost, which improves the agreement between the TMAP8 and analytical calculations.
```

Comment:
```suggestion
loading of 1 atom/m$^3$ in the first 10 m of a 100 m slab. TMAP4 and TMAP7 cases use a slab length of 2275 m, however using a smaller slab length was found to not change the results. Additionally, the smaller domain size allows getting a finer simulation mesh for the same computational cost, which improves the agreement between the TMAP8 and analytical calculations.
```

Reply:
I thought about this, but TMAP7 did not actually mention a slab length so I wanted to wait till I see the TMAP7 fortran scripts to be sure.

Reply:
^ I updated the suggestion.